app/uk/gov/hmrc/taxhistory/model/api/FillerState.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.taxhistory.model.api
+
+import uk.gov.hmrc.time.TaxYear
+
+trait FillerState
+
```

Comment:
Can we add some comments or scaladoc explaining the purpose of FillerState please

Reply:
Scaladoc added to trait and fillerState function, private methods described with comments.